runtime/src/iree/hal/drivers/amdgpu/registration/driver_module.c

```diff
@@ -0,0 +1,129 @@
+// Copyright 2025 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/hal/drivers/amdgpu/registration/driver_module.h"
+
+#include "iree/base/api.h"
+#include "iree/base/internal/flags.h"
+#include "iree/hal/drivers/amdgpu/api.h"
+
+IREE_FLAG_LIST(string, amdgpu_libhsa_search_path,
+               "Search path (directory or file) for the ROCR-Runtime library "
+               "(`libhsa-runtime64.so`, etc).");
+
+IREE_FLAG(
+    int64_t, amdgpu_host_block_pool_small_size, 0,
+    "Size in bytes of a small host block in the pool. Must be a power of two.");
+IREE_FLAG(
+    int64_t, amdgpu_host_block_pool_large_size, 0,
+    "Size in bytes of a large host block in the pool. Must be a power of two.");
+
+IREE_FLAG(int64_t, amdgpu_device_block_pool_small_size, 0,
+          "Size in bytes of a small device block in the pool. Must be a power "
+          "of two.");
```

Comment:
Should these flags default to something other than `0`? Or does `0` indicate "use the default"? Could explain that in the text. If I were setting "small" and "large" sizes in bytes, I might default to 64 or 1024 without any extra context... which might not be the right order of magnitude here.

Reply:
(will be in a fixup in #20990)